v0.19.0 can't be built by jitpack

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':fmi-native:compileReleaseLinuxX86-64Cpp'.
> No tool chain is available to build C++ for host operating system 'Linux' architecture 'x86-64':
    - Tool chain 'visualCpp' (Visual Studio):
        - Visual Studio is not available on this operating system.
    - Tool chain 'gcc' (GNU GCC):
        - Could not find C++ compiler 'g++' in system path.
    - Tool chain 'clang' (Clang):
        - Could not find C++ compiler 'clang++' in system path.

--- a/fmi-import/build.gradle
+++ b/fmi-import/build.gradle
@@ -24,5 +24,7 @@ shadowJar {
     archiveVersion = null
 }
 
-compileKotlin.dependsOn(':fmi-native:copyNativeLibs')
+if (!project.getProperties().get("skipBuildNative", false)) {
+    compileKotlin.dependsOn(':fmi-native:copyNativeLibs')
+}
 clean.dependsOn(copyTestFmus)

--- a/fmi-native/build.gradle
+++ b/fmi-native/build.gradle
@@ -5,67 +5,73 @@ plugins {
     id "cpp-library"
 }
 
-library { CppLibrary lib ->
+if (!project.skipBuildNative) {
 
-    baseName = "fmi2_jni"
+    library { CppLibrary lib ->
 
-    linkage = [Linkage.SHARED]
+        baseName = "fmi2_jni"
 
-    targetMachines = [
-        machines.windows.x86,
-        machines.windows.x86_64,
-        machines.linux.x86_64,
-    ]
+        linkage = [Linkage.SHARED]
 
-    lib.binaries.whenElementFinalized { CppBinary binary ->
-        project.dependencies {
+        targetMachines = [
+                machines.windows.x86,
+                machines.windows.x86_64,
+                machines.linux.x86_64,
+        ]
 
-            add(binary.includePathConfiguration.name, files("${org.gradle.internal.jvm.Jvm.current().javaHome}/include"))
+        lib.binaries.whenElementFinalized { CppBinary binary ->
+            project.dependencies {
 
-            if (OperatingSystem.current().isLinux()) {
-                add(binary.includePathConfiguration.name, files("${org.gradle.internal.jvm.Jvm.current().javaHome}/include/linux"))
-            } else if (OperatingSystem.current().isWindows()) {
-                add(binary.includePathConfiguration.name, files("${org.gradle.internal.jvm.Jvm.current().javaHome}/include/win32"))
+                add(binary.includePathConfiguration.name, files("${org.gradle.internal.jvm.Jvm.current().javaHome}/include"))
+
+                if (OperatingSystem.current().isLinux()) {
+                    add(binary.includePathConfiguration.name, files("${org.gradle.internal.jvm.Jvm.current().javaHome}/include/linux"))
+                } else if (OperatingSystem.current().isWindows()) {
+                    add(binary.includePathConfiguration.name, files("${org.gradle.internal.jvm.Jvm.current().javaHome}/include/win32"))
+                } else {
+                    throw new IllegalStateException("Unsupported OS: " + OperatingSystem.current().name)
+                }
+
             }
 
         }
 
     }
 
-}
+    def assembleAllRelease = []
 
-def assembleAllRelease = []
-
-tasks.all {
-    if (it.name.contains("assembleRelease")) {
-        assembleAllRelease.add(it)
+    tasks.all {
+        if (it.name.contains("assembleRelease")) {
+            assembleAllRelease.add(it)
+        }
     }
-}
 
-tasks.register("assembleAllRelease") {
-    dependsOn assembleAllRelease
-}
+    tasks.register("assembleAllRelease") {
+        dependsOn assembleAllRelease
+    }
 
-task copyNativeLibs(type: Copy, dependsOn: assembleAllRelease) {
+    task copyNativeLibs(type: Copy, dependsOn: assembleAllRelease) {
 
-    into "$rootDir/fmi-import/src/main/resources/native/fmi2"
+        into "$rootDir/fmi-import/src/main/resources/native/fmi2"
 
-    if (OperatingSystem.current().isLinux()) {
-        into ("linux64") {
-            from "$buildDir/lib/main/release/linux/x86-64"
-            include "*.so"
+        if (OperatingSystem.current().isLinux()) {
+            into("linux64") {
+                from "$buildDir/lib/main/release/linux/x86-64"
+                include "*.so"
+            }
+        } else if (OperatingSystem.current().isWindows()) {
+            into("win32") {
+                from "$buildDir/lib/main/release/windows/x86"
+                include "*.dll"
+            }
+            into("win64") {
+                from "$buildDir/lib/main/release/windows/x86-64"
+                include "*.dll"
+            }
+        } else {
+            throw new Exception("Unsupported OS: " + OperatingSystem.current().name)
         }
-    } else if (OperatingSystem.current().isWindows()) {
-        into ("win32") {
-            from "$buildDir/lib/main/release/windows/x86"
-            include "*.dll"
-        }
-        into ("win64") {
-            from "$buildDir/lib/main/release/windows/x86-64"
-            include "*.dll"
-        }
-    } else {
-        throw Exception("Unsupported OS: " + OperatingSystem.current().name)
+
     }
 
 }

--- a/fmi-native/build.gradle
+++ b/fmi-native/build.gradle
@@ -5,7 +5,7 @@ plugins {
     id "cpp-library"
 }
 
-if (!project.skipBuildNative) {
+if (!project.getProperties().get("skipBuildNative", false)) {
 
     library { CppLibrary lib ->
 

--- a/jitpack.yml
+++ b/jitpack.yml
@@ -1,2 +1,2 @@
 install:
-  - gradlew -PskipBuildNative=true clean publishToMavenLocal
+  - ./gradlew -PskipBuildNative=true clean publishToMavenLocal

--- a/jitpack.yml
+++ b/jitpack.yml
@@ -1,0 +1,2 @@
+install:
+  - gradlew -PskipBuildNative=true clean publishToMavenLocal

